src/userdel.c

```diff
@@ -1150,7 +1156,7 @@ int main (int argc, char **argv)
 	 * Note: This is a best effort basis. The user may log in between,
 	 * a cron job may be started on her behalf, etc.
 	 */
-	if (user_busy (user_name, user_id) != 0) {
+	if (!Rflg && user_busy (user_name, user_id) != 0) {
```

Comment:
Hi - I'm fine merging as-is, but just to make sure - do you want to check whether the user_name is busy in the chroot?

Reply:
No, I don't. It would be basically impossible to do.